src/builtins/paginate.rs

```diff
@@ -0,0 +1,91 @@
+//! Sample pagination implementation
+
+use crate::serenity_prelude as serenity;
+
+/// This is an example implementation of pagination. To tweak the behavior, copy the source code and
+/// adjust to your needs:
+/// - change embed appearance
+/// - use different emojis for the navigation buttons
+/// - add more navigation buttons
+/// - change timeout duration
+/// - add a page selector dropdown
+/// - use reactions instead of buttons
+/// - remove message after navigation timeout
+/// - ...
+///
+/// Note: this is a long-running function. It will only return once the timeout for navigation
+/// button interactions has been reached.
+///
+/// # Example
+///
+/// ```rust,no_run
+/// # async fn _test(ctx: poise::Context<'_, (), serenity::Error>) -> Result<(), serenity::Error> {
+/// let pages = &[
+///     "Content of first page",
+///     "Content of second page",
+///     "Content of third page",
+///     "Content of fourth page",
+/// ];
+///
+/// poise::samples::paginate(ctx, pages).await?;
+/// # Ok(()) }
+/// ```
+///
+/// ![Screenshot of output](https://i.imgur.com/JGFDveA.png)
+pub async fn paginate<U, E>(
+    ctx: crate::Context<'_, U, E>,
+    pages: &[&str],
+) -> Result<(), serenity::Error> {
+    // Define some unique identifiers for the navigation buttons
+    let ctx_id = ctx.id();
+    let prev_button_id = format!("{}prev", ctx.id());
+    let next_button_id = format!("{}next", ctx.id());
+
+    // Send the embed with the first page as content
+    let mut current_page = 0;
+    ctx.send(
+        crate::CreateReply::default()
+            .embed(serenity::CreateEmbed::default())
```

Comment:
```suggestion
            .embed(serenity::CreateEmbed::default().description(pages[current_page]))
```

It has no description, so the message can't even send, this will send the first page.